hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCallTransitionLogic.java

```diff
@@ -94,10 +94,10 @@ public void doStateTransition() {
 		// --- Translate from gRPC types ---
 		var contractCallTxn = txnCtx.accessor().getTxn();
 		final var senderId = Id.fromGrpcAccount(contractCallTxn.getTransactionID().getAccountID());
-		doStateTransitionOperation(contractCallTxn, senderId);
+		doStateTransitionOperation(contractCallTxn, senderId, false);
 	}
 
-	public void doStateTransitionOperation(final TransactionBody contractCallTxn, final Id senderId) {
+	public void doStateTransitionOperation(final TransactionBody contractCallTxn, final Id senderId, boolean isEthTx) {
```

Comment:
Rather than tying this to the ethereum transaction I think the field should be "incrementsNonce" or "incrementsCounter"

```suggestion
	public void doStateTransitionOperation(final TransactionBody contractCallTxn, final Id senderId, boolean incrementCounter) {
```

Reply:
done